src/odemis/acq/move.py

```diff
@@ -643,6 +643,18 @@ def _doCryoSwitchSamplePosition(future, target):
                     if future._task_state == CANCELLED:
                         logging.info("Cancelling aligner movement...")
                         raise CancelledError()
+
+                # If the stage rotates a lot, move it first to a safe position (Moving it in X and RZ is sufficient)
+                if abs(current_pos["rx"] - target_pos[target]["rx"]) > numpy.deg2rad(2):
```

Comment:
Please use the standard `math.radians()` function instead of the numpy one (we typically use numpy functions only when it needs to handle numpy arrays).